### ISSUE TITLE
The window to load a replay differs a lot when open from main menu and in game, keyboard issues

### ISSUE THREAD

**Issue by frankystone** [widelands/widelands#6578]
If you load a replay from the main menu it looks like this:
![replay_menu_from_main_menu](https://github.com/user-attachments/assets/5dd6bedf-2de6-4945-a157-bd77283e8817)

If you load a replay from the in game menu it looks like this:
![replay_menu_from_in_game](https://github.com/user-attachments/assets/cf82954d-c259-462f-8f18-9b2654fea80e)

Additionally when loading from in game one can't choose an entry with arrow keys. Marking one with the mouse and hitting ↓ key activates the table. Pressing the ↓ key several times the first entry is marked, then the second entry and then the table again.

**To reproduce**
1. click "load replay" from  the main menu
2. chose a replay, all works fine, arrow keys will work
3. start replay
4. click "load replay" from the in game main menu
5. there is less info in this screen
6. you have to click twice to choose an entry
7. arrow keys do not work

**Expected behavior**
Both windows should be equal in information and usability.

**Version:**
 - OS: archlinux
 - Widelands Version: "v1.2.1", and 1.3~git26808 (218fb49@master) Debug
 - Enabled Add-Ons: minimalistic theme

**Additional context**
I thought there was similar bug already, but i didn't found it.

**Comment by bunnybot** [comment]
Mirrored on Codeberg as [#CB4938](https://codeberg.org/wl/widelands/issues/4938).

**Comment by gpalino** [comment]
This one is similar bug: #6321

**Comment by bunnybot** [comment]
Assigned to Nordfriese
